src/common/SmartNumericInput/SmartNumericInput.tsx

```diff
@@ -49,6 +50,7 @@ export class SmartNumericInput extends React.Component<SmartNumericInputProps, {
                         />
                     )}
                 </div>
+                {this.props.originalValue !== this.props.value && <label>Orginalt: {this.props.originalValue}</label>}
```

Comment:
This kind of calculation should probably be done in the scope of the calling function (render) rather than in the actual return.

Reply:
I agree, it would be tidier.